bin/install.sh

```diff
@@ -67,7 +67,7 @@ for package in "${packages[@]}"; do
         sudo dnf config-manager --add-repo $repo_url
 
         # install the package
-        sudo dnf install terraform -y
+        sudo dnf install terraform-1.9.8-1 -y
```

Comment:
Wouldn't some error checking be wise?  The same would go for the above `dnf install` line.

Reply:
Adding onto this, it might be wise to have the versions recorded off in a different file (so we can source it and use $TF_DNF_VERSION or something).

Makes it easier to bump versions in the future if needed

Reply:
I've added the error checking in. 

I think the log file is a good idea - in the interest of keeping the scope of this PR focused I've opened a new issue for it (#137).